docs/deployment/manual_app_registration.md

```diff
@@ -0,0 +1,92 @@
+# App Registration Creation Guide
```

Comment:
How will someone find this documentation? This page is just in a random folder. I recommend linking to it from the Optional Features page.

Reply:
Link back to deployment procedure .

Reply:
Completed